docs/contents/components/forms/toggle/accessibility.ja.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: Toggle
+description: "`Toggle`は、オンまたはオフの２つの状態を持つコンポーネントです。"
+package_name: "@yamada-ui/toggle"
+order: 3
+with_description: true
+tab: アクセシビリティ
+---
+
```

Comment:
As with [Tabs](https://yamada-ui.com/components/disclosure/tabs/accessibility), for icon-only display, please include a description of the `aria-label` to be added.

Reply:
Fix.
https://github.com/yamada-ui/yamada-ui/pull/3073/files#diff-ace09ac64909f6671c2c20b32ef74d702fc8e7a94b4a6002e3764dd60f4160bbR10-R16

https://github.com/yamada-ui/yamada-ui/pull/3073/files#diff-42ea886f23e000338f5e35247d98e8da5d573c13bdd281441b81c56921164005R10-R16